azure-pipelines.yml

```diff
@@ -0,0 +1,66 @@
+pool:
+  name: OETest-OfficeClientApps
+
+steps:
```

Comment:
I might miss the point, but most of the code here is exactly equal to the code in the other files.
Is there a way to import the code from them? Otherwise, there will be a lot of duplicate code.

Reply:
Test case leftovers before I split it up for reuse.  Should have been deleted.  Will correct.